### PR TITLE
fix cpu and memory assignment for libvirt based boxes

### PR DESCRIPTION
**Pull request by Toasterson**
This commit got forgotten in my git branches :) fixes memory assignments

### CODE REVIEW

**Review by AndWac — APPROVED** [review]

File: Vagrantfile [hunk 1]
```diff
@@ -36,28 +36,28 @@ Vagrant.configure("2") do |config|
...
-    elsif host =~ /linux/
-      # meminfo shows size in kB; convert to Bytes
-      mem = `awk '/MemTotal/ {print $2}' /proc/meminfo`.to_i * 1024
-      cpus = `getconf _NPROCESSORS_ONLN`.to_i
-    elsif host =~ /mswin|mingw|cygwin/
-      # Windows code via https://github.com/rdsubhas/vagrant-faster
-      mem = `wmic computersystem Get TotalPhysicalMemory`.split[1].to_i
-      cpus = `echo %NUMBER_OF_PROCESSORS%`.to_i
-    end
+	# Get memory size and CPU cores amount
```
Comment (pyhalov) [inline Vagrantfile]: Indentation seems inconsistent. Should these lines be on `host` level?

Comment (Toasterson) [reply Vagrantfile]: I fixed indentation file wide. It was all over the place.

**Comment by pyhalov** [comment]
Please, resolve conflicts

**Comment by Toasterson** [comment]
Done

**Comment by pyhalov** [comment]
Github still complains 'This branch cannot be rebased due to conflicts'

**Comment by Toasterson** [comment]
Not for me. Try Control Reloading all the page?

**Comment by pyhalov** [comment]
No luck. Please merge latest changes from OpenIndiana/oi-userland oi/hipster branch to your oi/hipster branch, rebase Toasterson:vagrant-libvirt branch on oi/hipster and push both branches to your repository.

**Comment by Toasterson** [comment]
Huh, nasty. Done with your process.

**Pull request merged**